tests/wicked/aggregate/sut/t16_teaming_lacp.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, LACP
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use testapi;
+use network_utils 'ifc_is_up';
+
+sub run {
+    my ($self, $ctx) = @_;
+    record_info('INFO', 'Teaming, LACP');
+    $self->setup_team('lacp', $ctx->iface(), $ctx->iface2());
+    validate_script_output('wicked ifstatus --verbose all|grep ' . $ctx->iface(),  sub { /enslaved/ });
```

Comment:
1. I understand why you not using `validate_interfaces`  ( don't want to die when team0 is not up ) but I don't understand why you do same checks differently ( `wicked ifstatus`  here vs `ip a` within `validate_interfaces` ). Not saying that it is wrong , other way around actually . Maybe we need to do the same in `validate_interfaces`  ?
2. also maybe it would be better to do ifstatus for certain device instead of doing it for all and then grep needed ?

Reply:
ok, I was just following Marius comment about making sure that the interfaces are enslaved, but I guess that's the same thing as `'master eth0'` in validate interfaces